### PR TITLE
Feature/condo/doma 1400/post service subscription payment to sbbol

### PR DESCRIPTION
**Pull request by AntonAL**
Posts unsigned payment request to SBBOL without `crucialFieldsHash`.

Further to implement:

- [ ] Sign payment request with signature, obtained from SBBOL representatives
- [ ] Calculate hash for digested data
- [x] Recognize error response
- [x] Track successful post request to Fintech API in `ServiceSubscriptionPayment` record
- [x] Store successful response in `ServiceSubscriptionPayment` record
- [x] Store error response in `ServiceSubscriptionPayment` record
- [x] Implement lifecycle of `ServiceSubscriptionPayment` record, which have error response from SBBOL

## Testing on localhost
There is a page for that in Confluence

### CODE REVIEW

File: apps/condo/domains/subscription/utils/testSchema/index.js [hunk 1]
```diff
@@ -75,9 +82,73 @@ const expectOverlappingFor = async (action, ...args) => (
...
+async function createTestServiceSubscriptionPayment (client, organization, subscription, extraAttrs = {}) {
+    if (!client) throw new Error('no client')
+    if (!organization || !organization.id) throw new Error('no organization.id')
+    if (!subscription || !subscription.id) throw new Error('no subscription.id')
+    const sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+
+    const type = 'default'
+    const status = SUBSCRIPTION_PAYMENT_STATUS.CREATED
+    const amount = faker.datatype.float({ precision: 0.01 }).toString()
+    const currency = 'RUB'
```
Comment (Dimitreee) [inline apps/condo/domains/subscription/utils/testSchema/index.js:94]: Sync currencies with @toplenboren and @SavelevMatthew

File: apps/condo/domains/common/utils/parallel.test.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+const { executeInSequence, processArrayOf } = require('./parallel')
```
Comment (pahaz) [inline apps/condo/domains/common/utils/parallel.test.js:1]: parallel.test.js -> parallel.spec.js

Comment (AntonAL) [reply apps/condo/domains/common/utils/parallel.test.js]: Changed

File: apps/condo/domains/organization/tasks/syncSbbolPaymentRequestsForSubscriptions.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const { createCronTask } = require('@core/keystone/tasks')
+const { syncPaymentRequestsForSubscriptions } = require('../integrations/sbbol/sync/syncPaymentRequestsForSubscriptions')
+
+/**
+ * Checking of SBBOL payment requests should be performed more frequently
+ */
+const task = createCronTask('syncSbbolPaymentRequestsForSubscriptions', '* * * * *', async () => {
```
[the remark this answers is not in the document]
Comment (AntonAL) [reply apps/condo/domains/organization/tasks/syncSbbolPaymentRequestsForSubscriptions.js]: Changed

File: apps/condo/domains/organization/integrations/sbbol/sync/syncSubscriptions.js [hunk 4]
```diff
@@ -111,15 +122,14 @@ const syncSubscriptions = async ({ context, date }) => {
     const fintechApi = new SbbolFintechApi(ourOrganizationAccessToken)
     debugMessage('Checking, whether the user have ServiceSubscription items')
 
-    const advanceAcceptances = await fintechApi.fetchAdvanceAcceptances({ date, clientId: SBBOL_CONFIG.client_id })
+    const advanceAcceptances = await fintechApi.fetchAdvanceAcceptances({ date: '2021-10-08', clientId: SBBOL_CONFIG.client_id })
```
Comment (pahaz) [inline apps/condo/domains/organization/integrations/sbbol/sync/syncSubscriptions.js:125]: Need to fix it

**Pull request merged**